examples/todo-app/webpack.config.js

```diff
@@ -0,0 +1,53 @@
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+const path = require("path");
+const APP_PATH = path.resolve(__dirname, "src");
+
+module.exports = {
+  entry: APP_PATH,
+
+  output: {
+    filename: "bundle.js",
+    path: path.resolve(__dirname, "dist"),
+    publicPath: "/",
+  },
+
+  resolve: {
+    extensions: [".ts", ".tsx", ".js", ".json"],
+  },
+
+  devtool: "source-map",
+
+  module: {
+    rules: [
+      {
+        test: /\.(ts|js)x?$/,
+        use: ["babel-loader", "source-map-loader"],
+        exclude: /node_modules/,
+      },
+      {
+        test: /\.scss$/,
+        loader: ["style-loader", "css-loader", "sass-loader"],
+      },
+      {
+        test: /\.css$/i,
+        use: ["style-loader", "css-loader"],
+      },
+      {
+        test: /\.(png|jpe?g|svg|ico)$/,
+        loader: "file-loader",
+        options: {
+          publicPath: "/",
```

Comment:
Do we need both this `publicPath` and the `publicPath` above?

Reply:
I think so - will try and remove and see